templates/degr_peak_calling.R

```diff
@@ -4,6 +4,11 @@ library(magrittr)
 library(glue)
 library(parallel)
 library(assertthat)
+
+
+setDTthreads(1)
+
```

Comment:
So here ste to $param.cores, before cluster set to 1 and after cluster back to param cores